### ISSUE TITLE
Broken coming soon template in dataview

### ISSUE THREAD

**Issue by ilyasfoo** [woothemes/woocommerce#49699]
### Issue Description

Originally reported in p1719905292036319-slack-C01SFMVEYAK and p1721309784520059-slack-C06DJHTE04U

The coming soon entire site thumbnail in templates dataview is broken.

![image](https://github.com/user-attachments/assets/5c8e9348-b098-41d2-8d9b-1b41591a60b8)

The cause might have been because we set `vh` to `width` and `height` of the template component.

### Acceptance Criteria

We're not sure what's the most appropriate fix, but Gutenberg upstream fix will be the most ideal. Otherwise, there's a couple potential workarounds:

1. Dynamically change to `%` width & height if it’s rendered from a dataview
2. Amend the design so it doesn't depend on having to use `vh`, but the template needs to stretch to fill the screen both horizontally and vertically

**Comment by adrianduffell** [comment]
> Gutenberg upstream fix will be the most ideal

I'd agree this is ideal, although we may also need a temporary fix on the Woo side until the next WP release. If so, it would be great to have a fix in before the 9.2 code freeze.

**Comment by ilyasfoo** [comment]
I've attempted to add the following to the template & styles to fix it, and it works if it's a fresh site:

```css
.block-editor-block-preview__content-iframe .coming-soon-cover {
    min-height: 1200px !important;
}
```

<img width="740" alt="image" src="https://github.com/user-attachments/assets/3dcc64e2-7993-440b-aa39-b36a52c5a85d">


However, if there's already a customization of the coming soon template, it will break and requires "Attempt recovery" in order to fix it 😅 

https://github.com/user-attachments/assets/e1e14756-d7e6-4d2c-99e7-bed35efb0b82

Looks like it might not be straightforward to fix within the template itself.
